valohai_cli/commands/execution/run/frontend_command.py

```diff
@@ -184,6 +189,7 @@ def run(
     k8s_devices: List[str],
     k8s_device_none: bool,
     k8s_preset: Optional[str],
+    ssh: Optional[bool] = False,
```

Comment:
`ssh`, `sync` and `watch` are mutually exclusive (all of them are blocking processes), please make it so :)

Reply:
I doubt `None` is an option here:
```suggestion
    ssh: bool = False,
```